app/models.py

```diff
@@ -1,6 +1,5 @@
-from datetime import datetime, timezone
-
-from sqlalchemy import Boolean, Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy import (Boolean, Column, DateTime, ForeignKey, Integer, String,
```

Comment:
you could name your DB models

`ListSQL` or `ListDB` so importing them in another module is easier because you dont need to use from .. import List as ...`